types/findup-sync/findup-sync-tests.ts

```diff
@@ -1,6 +1,6 @@
 import findup = require('findup-sync');
 
-let str: string;
+let str: string | null = null;
 
 str = findup('foo');
```

Comment:
Would you mind migrating the tests to the `// $ExpectType ...` assertions instead of assignments? `$ExpectType` assertions are more precise than assignments.

Reply:
sure, amended! thx!